### PR TITLE
Add socket and render messages

### PR DESCRIPTION
**Pull request by condef5**
### This pull request includes:

* Add connection from socket.
* Show messages from socket
* Create message from form
* Send message using websocket

solved #21

### CODE REVIEW

File: public/channel.html [hunk 1]
```diff
@@ -21,7 +23,6 @@ <h4 class="user__name">Diego Cuevas</h4>
     </ul>
   </div>
   <div class="messages" id="listMessage">
```
Comment (test0n3) [inline public/channel.html]: Change to `<ul>`, messages are presented as list elements.

File: public/channel.html [hunk 2]
```diff
@@ -53,7 +54,7 @@ <h4 class="user__name">Diego Cuevas</h4>
       </symbol>
     </defs>
   </svg>
-
+  <script src="js/main.js"></script>
 </body>
 <script src="js/main.js"></script>
```
Comment (christopher77) [inline public/channel.html]: is it necessary to have two scripts?

Comment (condef5) [reply public/channel.html]: Not, this error was caused when i did rebase, now I solve it.

File: public/channel.html (hunk 1 above)
Comment (condef5) [reply public/channel.html]: Thanks :)

**Pull request merged**